.eslintrc.js

```diff
@@ -63,6 +64,11 @@ module.exports = {
             "ignoreEnums": true,
             "ignoreReadonlyClassProperties": true
         }],
+        "@typescript-eslint/no-restricted-imports": ["error", {
+            "patterns": [{
+                "group": ["**/../lib", "**/../src"]
+            }]
+        }],
```

Comment:
This lint rule seems really useful - would be nice to adopt it for Zowe Explorer & ZE API 😋